Empty `residues_` in `LinearRegression`: docstring updated

deepcharles opened the pull request:
Updated doc in the linear_model/base.py (class LinearRegression)
Issue #5313

glouppe commented:
Please chose an appropriate title for the PR, so that contributors can spot and review it.

TomDLT commented:
Your code is not PEP8 friendly.
See https://pypi.python.org/pypi/flake8 for easy spotting.

deepcharles committed:
Fixed the docstring (issue #5313)

    Empty "residues_" in "LinearRegression": documented the case when the "residues_" attribute is empty.

amueller commented:
LGTM. merging as minor (even though this is now deprecated)
